docs/docs/hive.md

```diff
@@ -747,18 +747,20 @@ To reference a metadata table the full name of the table should be used, like:
 
 Currently the following metadata tables are available in Hive:
 
-* all_data_files 
-* all_delete_files 
-* all_entries all_files 
```

Comment:
I split this one into two lines. All the other diff have been caused by removing extra trailing spaces.
https://github.com/apache/iceberg/pull/11902/files?w=1